src/Model/ModelData.php

```diff
@@ -205,6 +204,7 @@ public function getDynamicData(string $field): mixed
 
     public function setDynamicData(string $field, mixed $value): static
     {
+        $this->objCacheClear();
```

Comment:
We forgot to add this when implementing this method. If someone calls `$model->obj('MyDynamicData')` and then calls `$model->setDynamicData('MyDynamicData', 'my value')` the obj cache should _not_ give the stale value.